lib/monitoring/aws-rds/RdsClusterMonitoring.ts

```diff
@@ -147,15 +166,106 @@ export class RdsClusterMonitoring extends Monitoring {
       this.addAlarm(createdAlarm);
     }
 
+    for (const disambiguator in props.addSelectLatencyAlarm) {
+      const alarmProps = props.addSelectLatencyAlarm[disambiguator];
+      const createdAlarm = this.latencyAlarmFactory.addLatencyAlarm(
+        this.selectLatencyMetric,
+        LatencyType.P90,
```

Comment:
Should we open the door for more latency types?

Maybe we can add an evolution of `LatencyThreshold` that also accepts the type so you could ultimately do something like

```
addSelectLatencyAlarm: {
  Critical: {
    latencyType: LatencyType.P99,
    ...
  },
  Warning: {
    latencyType: LatencyType.P90,
    ...
  },
}
```